coralproject/talk
# Fix admin View Context view for comments with SYSTEM_WITHHELD status

## What does this PR do?

Currently when the admin `View context` link is clicked on a comment with status `SYSTEM_WITHHELD`, the comment stream fails to load due to a thrown error / unhandled case for this status type in component `InactiveCommentLabel`.

<img width="702" alt="uncaught error - unknown inactive status system_withheld" src="https://user-images.githubusercontent.com/1551632/38851140-d9447dd6-4257-11e8-80a4-faee5e4da3b1.png">

This PR adds this status type to the component to resolve the issue.

<img width="381" alt="view context - comment stream - comment status system_withheld" src="https://user-images.githubusercontent.com/1551632/38851160-ec6505d4-4257-11e8-9c1e-efc56110b1e2.png">

## How do I test this PR?

- Visit the admin
- Click the View context link for a comment with status `SYSTEM_WITHHELD`
- Confirm the comment stream loads successfully with the comment and new label (System Withheld) displayed
